lib/cc/engine/analyzers/javascript/main.rb

```diff
@@ -15,8 +15,9 @@ class Main < CC::Engine::Analyzers::Base
             "**/*.jsx"
           ]
           LANGUAGE = "javascript"
-          DEFAULT_MASS_THRESHOLD = 40
-          BASE_POINTS = 3000
+          DEFAULT_MASS_THRESHOLD = 28
```

Comment:
sure that sounds good. 

In general, it felt to me like there were opportunities to refactor in this code, but I wanted to avoid additional changes in this PR and refactor separately.

I'm happy with the threshold one though, adding that now!